packages/tamagotchi/src/main/index.ts

```diff
@@ -0,0 +1,99 @@
+import { readFileSync } from 'node:fs'
+import { join } from 'node:path'
+import process from 'node:process'
+import { electronApp, is, optimizer } from '@electron-toolkit/utils'
+import { app, BrowserWindow, ipcMain, shell } from 'electron'
+import icon from '../../resources/icon.png?asset'
```

Comment:
```suggestion
import icon from '../../build/icon.png?asset'
```

After this, `packages/tamagotchi/resources/icon.png` should be deleted.